packages/patternfly-4/src/pages/index.js

```diff
@@ -1,20 +1,215 @@
+      <Grid>
+        <GridItem sm={12} md={6} mdOffset={3} className="pf-u-text-align-center">
+          <TextContent>
+            <img src={orb} alt="PatternFly logo" />
+            <Title size="4xl" className="pf-m-white">Build better experiences with repeatable, scalable design.</Title>
+            <Title size="xl" className="pf-m-white pf-u-mb-md pf-u-mb-3xl-on-md">PatternFly 4 is an open source design system built to drive consistency and unify teams.</Title>
+            <Link to="/get-started/about">
+              <button type="button" aria-label="Get started with PatternFly" className="pf4-c-button__cta-primary pf-u-mr-0 pf-u-mr-sm-on-sm pf-u-mb-md pf-u-mr-xl-on-md">Get Started</button>
+            </Link>
+            <Link to="#AboutPatternFly4">
+              <button type="button" className="pf4-c-button__cta-outline" aria-label="Why PatternFly 4">Why PatternFly 4</button>
+            </Link>
+          </TextContent>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection variant={PageSectionVariants.light}>
+      <Grid>
+        <GridItem sm={12} md={4} mdOffset={4} className="pf-u-text-align-center">
+          <Title size="4xl" id="AboutPatternFly4" className="pf-u-mb-md">About PatternFly 4</Title>
+          <Title size="lg">We provide tools like design documentation, components, and code examples to make it possible for anyone to design and build responsive, accessible user experiences.</Title>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection variant={PageSectionVariants.light} className="pf4-c-background-lines">
+      <Grid gutter="md" className="pf-u-pb-3xl">
+        <GridItem sm={12}>
+          <Grid>
+            <GridItem sm={12} md={4} mdOffset={2}>
+              <TextContent>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Make better decisions with clear guidance</Text>
+                  <Text component={TextVariants.p}>Design guidelines supply the foundation of the system. Get acquainted with our approach to icons, color, content, and more.</Text>
+                  <Text component={TextVariants.a} href="/design-guidelines/styles/icons" aria-label="view design guidelines" className="pf-u-pl-0 pf4-c-link__text">View design guidelines <CaretRightIcon />
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Start building with components</Text>
+                  <Text component={TextVariants.p}>Components are like building blocks. Designed to be flexible and modular, you can mix and match to create a solution for almost any UI problem.</Text>
+                  <Text component={TextVariants.a} href="/documentation/react/components/alert" aria-label="view components" className="pf-u-pl-0 pf4-c-link__text">View components <CaretRightIcon />
```

Comment:
These links are react specific, are we good with that or need to provide links to react and html/core?

Reply:
I'm good with the react specific links, as that is what we are surfacing first in most cases. Worth checking with @rachael-phillips and @stacymcauliffe on.

Reply:
Yep, this is what we expect. Thanks, guys!